client/src/app/project-registration/project-registration-form/project-registration-form.component.ts

```diff
@@ -226,4 +235,25 @@ export class ProjectRegistrationFormComponent implements OnInit {
         concatMap(createdProject => this.ingestService.partiallyPatchProject(createdProject, this.patch))
       );
   }
+
+  onFormValueChange(formData: Observable<object>) {
+    formData.pipe(
```

Comment:
I would pack this logic into a properly named function, such as` `aveProjectInCache`